### ISSUE TITLE
TypeError: process.stdout.getWindowSize is not a function

### ISSUE THREAD

**Issue by zachlysobey** [alexjoverm/typescript-library-starter#137]
I successfully cloned the repo, but an `npm install` results in the error pasted below.

This is using `Git Bash`, Windows 10, npm 5.5.1, and node 8.9.1

```
let lines = (process.stdout as any).getWindowSize()[1]
                                    ^
TypeError: process.stdout.getWindowSize is not a function
    at Object.<anonymous> (C:\Users\zachl\workspace\framework\tools\init.ts:68:37)
    at Module._compile (module.js:635:30)
    at Module.m._compile (C:\Users\zachl\workspace\framework\node_modules\ts-node\src\index.ts:392:23)
    at Module._extensions..js (module.js:646:10)
    at Object.require.extensions.(anonymous function) [as .ts] (C:\Users\zachl\workspace\framework\node_modules\ts-node\src\index.ts:395:12)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Function.Module.runMain (module.js:676:10)
    at Object.<anonymous> (C:\Users\zachl\workspace\framework\node_modules\ts-node\src\_bin.ts:182:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! --libraryname--@0.0.0 postinstall: `ts-node tools/init`
```

**Comment by gimenete** [comment]
I think it would be better to just use `process.stdout.write('\x1B[2J\x1B[0f')`

**Comment by gimenete** [comment]
I made a PR here #138 In the meantime maybe you want to use my repo that contains that fix.

https://github.com/gimenete/typescript-library-starter/tree/master

**Comment by alexjoverm** [comment]
Fixed by #138
